packages/plugin-essentials/sources/commands/unlink.ts

```diff
@@ -0,0 +1,78 @@
+import {BaseCommand, WorkspaceRequiredError}                      from '@yarnpkg/cli';
+import {Cache, Configuration, Project, StreamReport, structUtils} from '@yarnpkg/core';
+import {npath, ppath}                                             from '@yarnpkg/fslib';
+import {Command, Option, Usage, UsageError}                       from 'clipanion';
+
+// eslint-disable-next-line arca/no-default-export
+export default class UnlinkCommand extends BaseCommand {
+  static paths = [
+    [`unlink`],
+  ];
+
+  static usage: Usage = Command.Usage({
+    description: `disconnect the local project from another one`,
+    details: `
+      This command will remove any resolutions in the project-level manifest that would have been added via a yarn link with similar arguments.
+    `,
+    examples: [[
+      `Unregister a remote workspace in the current project`,
+      `$0 unlink ~/ts-loader`,
+    ], [
+      `Unregister all workspaces from a remote project in the current project`,
+      `$0 unlink ~/jest --all`,
+    ]],
+  });
+
+  all = Option.Boolean(`-A,--all`, false, {
+    description: `Unlink all workspaces belonging to the target project from the current one`,
+  });
+
+  destination = Option.String();
+
+  async execute() {
+    const configuration = await Configuration.find(this.context.cwd, this.context.plugins);
+    const {project, workspace} = await Project.find(configuration, this.context.cwd);
+    const cache = await Cache.find(configuration);
+
+    if (!workspace)
+      throw new WorkspaceRequiredError(project.cwd, this.context.cwd);
+
+    const absoluteDestination = ppath.resolve(this.context.cwd, npath.toPortablePath(this.destination));
+
+    const configuration2 = await Configuration.find(absoluteDestination, this.context.plugins);
```

Comment:
```suggestion
    const configuration2 = await Configuration.find(absoluteDestination, this.context.plugins, {useRc: false, strict: false});
```

See https://github.com/yarnpkg/berry/pull/2831